src/_delete-bucket-contents.js

```diff
@@ -6,33 +6,52 @@ module.exports = function deleteBucketContents ({ bucket }, callback) {
   let region = process.env.AWS_REGION
   let s3 = new aws.S3({ region })
 
-  waterfall([
-    function (callback) {
-      s3.listObjectsV2({ Bucket: bucket }, function done (err, result) {
-        if (err) {
-          callback(err)
-        }
-        else if (result.IsTruncated) {
-          throw Error('bucket has too many objects to delete')
+  let objects = []
+  function collectObjects (ContinuationToken, callback) {
+    s3.listObjectsV2({
+      Bucket: bucket,
+      ContinuationToken
```

Comment:
Is this ok to be `null`? (I assume so but I feel like I never know with various aws-sdk methods...

Reply:
Yes! I tested this out on a live bucket and it seems to have worked fine.